### PR TITLE
ChibiBuddy tracking

### PR DESCRIPTION
**Pull request by gausie**
Goals:
- [x] Tracks chibichanged in prefs
- [x] Tracks chibi name in prefs
- [x] Tracks adventure-costing adjustments in prefs
- [x] Tracks chibi buddy stats in prefs
- [x] Tracks chibi buddy age in prefs
- [x] Reset everything on death
- [x] Track whether the chibi buddy's overnight activities have been observed yet today

Coming in a future PR:
- CLI command for controlling the chibibuddy

### CODE REVIEW

File: test/internal/helpers/Player.java [hunk 1]
```diff
@@ -292,6 +292,16 @@ public static Cleanups withItems(final int... itemIds) {
     return cleanups;
   }
 
+  /**
+   * Ensures that a given item is not in the player's inventory
+   *
+   * @param itemId Item to not have
+   * @return Restores the number of this item to zero
```
Comment (midgleyc) [inline test/internal/helpers/Player.java:299]:
```suggestion
   * @return Restores the number of this item to the old value
```

Comment (gausie) [reply test/internal/helpers/Player.java]: Confused myself there

**Review by midgleyc — APPROVED** [review]
Not sure what inspired this, but looks good :)

**Comment by gausie** [comment]
> Not sure what inspired this, but looks good :)

I use the familiar weight buff every day and would love if someone wrote a good tool for keeping them alive longer so I get more turns of my buff :)

**Pull request merged**